spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -33,7 +33,7 @@ A higher `ordinal` means that the values taken from this `ConfigSource` will ove
 This allows a configuration to be customized from outside a binary, assuming that external `ConfigSource` s have higher `ordinal` values than the ones whose values originate within the release binaries.
 
 It can also be used to implement a drop-in configuration approach.
-Simply create a jar containing a `ConfigSource` with a higher ordinal and override configuration values in it.
+Simply create a jar containing a `ConfigSource` with a higher ordinal and override or clear configuration values in it.
```

Comment:
Should we also add here that the empty string is considered an empty value for effects of clearing a configuration?